azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/sync_clients.py

```diff
@@ -218,13 +159,12 @@ class ModuleClient(GenericClientSync):
 
     def __init__(self, transport):
         super(ModuleClient, self).__init__(transport)
-        self._transport.on_transport_input_message_received = self._handle_input_message_received
+        self._transport.on_transport_input_message_received = (
+            self._message_queues.receive_input_message
```

Comment:
>receive_input_message [](start = 33, length = 21)

nit: naming: that sounds more like a "handle_input_message" or "enqueue_input_message"